api/URLSearchParams.json

```diff
@@ -29,9 +29,20 @@
             },
             {
               "version_added": "7.5.0",
-              "version_removed": "10.0.0",
               "partial_implementation": true,
-              "notes": "Only available implicitly or as import from the <code>URL</code> module."
+              "notes": "Available as a part of the <code>url</code> module."
+            },
+            {
+              "version_added": "7.0.0",
+              "version_removed": "7.5.0",
+              "partial_implementation": true,
+              "notes": "Only available as an internal instance."
+            },
```

Comment:
though the interface is not exposed directly, it can be accessed via `URL.prototype.searchParams` (i.e.  new url.URL('https://github.com/mdn/browser-compat-data/pull/24998').searchParams.__proto__.constructor is the equal call to `URLSearchParams`)

Reply:
It would be great if the note explains what the developer must do to access the instance instead, like ``Only accessible via `URL.searchParams`.``.  Could you rewrite the note accordingly (same for the other PRs)?

Reply:
I agree with Vinyl and made a similar suggestion in https://github.com/mdn/browser-compat-data/pull/25007#discussion_r1854577413.